smac/optimizer/smbo.py

```diff
@@ -14,7 +14,9 @@
 from smac.optimizer import pSMAC
 from smac.optimizer.acquisition import AbstractAcquisitionFunction
 from smac.optimizer.ei_optimization import InterleavedLocalAndRandomSearch, \
-    AcquisitionFunctionMaximizer, RandomSearch, ChooserNoCoolDown, ChooserLinearCoolDown
+    AcquisitionFunctionMaximizer, RandomSearch
+from smac.optimizer.random_configuration_chooser import ChooserNoCoolDown, \
+    ChooserLinearCoolDown
```

Comment:
Is there a reason to always implement ChooserLinearCooldown as well? It is not used by default, right?

Reply:
It is not used by default, but can be used via a command line switch so it has to be implemented as well.